src/Curl.CommandLine.Parser/Curl.CommandLine.Parser.csproj

```diff
@@ -0,0 +1,35 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <PackageId>CurlParser</PackageId>
```

Comment:
I believe that we should set the PackageId to `Curl.CommandLine.Parser`

Reply:
Thanks for notifying, done.